features/support/env.rb

```diff
@@ -13,52 +13,24 @@
   @audit
   @commit-transactions
   @javascript
+  @no-txn
 ].freeze
 
 transactional = non_transactional.map {|t| "not #{t}" }
 
 Before(*transactional) do
-  Cucumber::Rails::Database.javascript_strategy = :transaction
-  Cucumber::Rails::Database.before_js if Cucumber::Rails::Database.autorun_database_cleaner
+  DatabaseCleaner.strategy = :transaction
 end
 
 Before non_transactional.join(' or ') do
-  Cucumber::Rails::Database.javascript_strategy = :truncation
-  Cucumber::Rails::Database.before_non_js if Cucumber::Rails::Database.autorun_database_cleaner
+  DatabaseCleaner.strategy = :truncation
```

Comment:
The `Cucumber::Rails::Database` is supposed to abstract the underlying `DatabaseCleaner`

I think we should not expose `DatabaseCleaner` here

Reply:
It does not. `Cucumber::Rails::Database.javascript_strategy` only affects `@javascript` tagged scenarios and nothing else. 

See https://github.com/cucumber/cucumber-rails/blob/561ee249c418f1c8cd7b629e7cf414442ce62709/lib/cucumber/rails/hooks/active_record.rb

I have another commit that allows switching between `truncation`, `transaction` with threading hack and `transaction` without threading hack but it is needlessly complicated and we don't need it. See 2a16ffbeab79b9258a200b3d6dd8621e3223b7f7. Also see my cucumber/cucumber-rails#521 where I discuss this with upstream project.